app/static/playwright.config.ts

```diff
@@ -62,7 +62,7 @@ export default defineConfig({
      * Use the preview server on CI for more realistic testing.
      * Playwright will re-use the local server if there is already a dev-server running.
      */
-    command: process.env.CI ? 'npm run preview' : 'npm run dev',
+    command: process.env.CI ? 'npm run build' : 'npm run dev',
```

Comment:
So the server has already been run earlier in the workflow on CI, so we just need to build the front end?

Reply:
im actually not 100% sure how we are doing the playwright stuff yet, this is more of an inbetween change because preview builds the production bundle and then serves it but we dont want that in any situation because our express server first needs to inject some variables into it so but changed it to build, the server should be run after the build so that the bundle is copied to the public folder in the server before it starts up

Reply:
as a broader point ive also been wondering if its worth just changing to github actions for the browser tests, i think its worth it